### PR TITLE
fix(responsive-ui): Make modal full screen & fix prejoin layout on mobile landscape

### PR DESCRIPTION
**Pull request by vp8x8**
Before:

Screenshot 2021-11-16 at 11 08 14
Screenshot 2021-11-16 at 11 08 28

After:

Screenshot 2021-11-16 at 10 32 41
Screenshot 2021-11-16 at 11 04 32

### CODE REVIEW

File: css/_atlaskit_overrides.scss [hunk 1]
```diff
@@ -103,3 +103,20 @@ div.Tooltip {
     line-height: 14px;
     padding: 8px;
 }
+
+// make modal full screen on landscape orientation
+@media (max-height: 420px) {
+    .atlaskit-portal {
+        .css-1oc7v0j {
```
Comment (horymury) [inline css/_atlaskit_overrides.scss:110]: is this a static class name? I think using what to appears to be a generated class name is not safe.

**Review by quitrk — APPROVED** [review]

**Review by horymury — CHANGES_REQUESTED** [review]

File: css/_atlaskit_overrides.scss (hunk 1 above)
Comment (vp8x8) [reply css/_atlaskit_overrides.scss:110]: This is a generated class name. But as long as the library is not updated the name will be preserved (the class name will be generated with the same name). There is no other way to select that element given the dom structure. Also I've seen the use of such classes in the project.

Comment (saghul) [reply css/_atlaskit_overrides.scss:110]: Is there is a selector we can use instead of the name great, but if not, oh well, we've been in there before...

**Review by horymury — APPROVED** [review]

**Comment by vp8x8** [comment]
Jenkins please test this.

**Comment by vp8x8** [comment]
Jenkins test this please.

**Comment by vp8x8** [comment]
Jenkins test this please.

**Pull request merged**